content/docs/user-guide/basic-concepts/dependency.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Dependency
+match: [dependency, dependencies]
+---
+
+A file or directory (possibly tracked by DVC) recorded in the "deps" section of
```

Comment:
looks like in most cases we want single quotes not double - for inline code blocks